.github/workflows/trufflehog.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
+
+name: Secret Leaks
+
+jobs:
+  trufflehog:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
Is there a reason to fetch everything?

Reply:
Though we could make it so it goes through the whole history if we wanted, cf the actions code: https://github.com/trufflesecurity/trufflehog/blob/3a029ea1932f93109bf9310607ed0f426483a891/action.yml

Reply:
Shallow cloning sounds good!

About making it go through the whole history, pretty sure nothing's been merged with secrets ever, but we can run it once if we want (no need to run it every time)